docs/configuration.rst

```diff
@@ -0,0 +1,26 @@
+Configuration Reference
+=======================
+
+``towncrier`` has many knobs and switches you can use, to customise it to your project's needs.
+The setup in the `Quick Start <quickstart.html>`_ doesn't touch on many, but this document will detail each of these options for you!
+
+For how to perform common customisation tasks, see `Customisation <customisation/index.html>`_.
```

Comment:
Same or

    :doc:`Customization <customization/index>`

if you don't want to use the name of the headline.

Reply:
Or `ref:Customization` to avoid the `…/index`.